src/components/utils.ts

```diff
@@ -71,6 +71,13 @@ export const getAlertsAndRules = (
     return _.filter(g.rules, { type: 'alerting' }).map(addID);
   });
 
+  // Add external labels to all `rules[].alerts[].labels`
+  rules.forEach((rule) => {
+    if (rule.alerts.length > 0) {
```

Comment:
I think the `if (rule.alerts.length > 0)` should be redundant since the `forEach` will do nothing for an empty array anyway?

Reply:
@zhuje do we need to update code based on comment from @kyoto ?

Reply:
Updated in the latest commit.

Reply:
@juzhao would you be able to test one more time, please?

Reply:
@zhuje tested again, if this PR is only for admin console only, no issue, but for developer console, there is one bug [OU-252](https://issues.redhat.com/browse/OU-252), maybe we have not implement this feature to developer console

Reply:
Thanks @juzhao. This PR just affects the admin console, so it is expected that this doesn't add external labels to the developer console. We plan to transition the dev console Alerting page UX to match the admin console, which will resolve this.